deployment-examples/kubernetes/kubeapply

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+if [ ! -f ".env" ]; then
+   echo ".env file does not exist. Please create .env file containing the variables."
+   exit
+fi
+
+CREATE=false
+FILE=${1}
+
+if [ $1 ] && [ $1 == "--create" ]; then
+   CREATE=true
```

Comment:
After looking at this, I think it's best if instead of doing kubectl apply / kubectl create, the script can simply in-place edit the file and insert the changes. That's all, user can manually perform `kubectl` actions as needed! :smile: